### ISSUE TITLE
External keyboard focus indicator appearance

### ISSUE THREAD

**Issue by TanyaVanWorkum** [w3c/matf#97]
In our audits, when we test the app with an external keyboard, we check whether the keyboard focus indicator meets 2.4.7, 1.4.1, and 1.4.11. We perform these tests using an external keyboard only.

For native iOS, there is a setting under Accessibility > Keyboards and Typing > Full Keyboard Access On > Appearance. Under Appearance, it is possible to change the color of the keyboard focus indicator and enable high contrast. If high contrast is turned on, an external keyboard focus indicator would pass 2.4.7, 1.4.1, and 1.4.11.

If high contrast is not turned on, the keyboard focus indicator will not always have an additional cue, such as an extra focus frame (similar to the screen reader focus indicator). Often, it appears as a sort of frame filled with color. Sometimes, it may be barely visible, depending on the background color and the color of the interactive elements. However, since it is possible to change the appearance of the indicator in the settings, it will still pass 1.4.1 and 1.4.11.

However, in Flutter, an external keyboard on iOS works only when Full Keyboard Access is turned off, meaning that the appearance of the focus indicator cannot be adjusted in the settings. In these cases, would the keyboard focus indicator fail 1.4.1 if there is no additional cue beyond the filled color of the focus indicator? And what if the keyboard focus indicator in Flutter has not been modified by developers? Would it still pass, even though it is barely visible and/or does not have an additional cue like an extra frame? Curious to hear thoughts on this.

**Comment by detlevhfischer** [comment]
Hi @TanyaVanWorkum 
This is our current take, and I would also welcome more input on that. 

- If no keyboard focus is visible, this clearly fails (11.)2.4.7. 
- If the keyboard focus is visible at all, but contrast not sufficient, we would pass that due to the exception in 1.4.11:
>"where the appearance of the component is determined by the user agent and not modified by the author".
